### ISSUE TITLE
Screen reader is not announcing the user entered bullet point in Organization Chart.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15113]
### Steps to reproduce:
1. Open Word document and press Alt+N+M keys.
2. Navigate to “Hierarchy” and select Organization Chart option.
3. Create a chart.
4. Press tab to validate if the user defined name(s) are being announced or not.
5. Verify if screen reader is announcing the style names or not.

### Actual behavior:
Screen reader is not announcing the user entered text. Screen Reader is just announcing the level and position of the bullet. 

### Expected behavior:
Screen reader should be announcing the user entered text along with the level and position of the bullet.

#### Windows version: Edition: Windows 11 Enterprise
Version	22H2
OS build	22621.1848
Experience: Windows Feature Experience Pack 1000.22642.1000.0

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by Adriani90** [comment]
@msftedad can you reprotuce this issue with narator as well?
